Readme.md

```diff
@@ -104,23 +98,15 @@ how to use Infogami (and its database, Infobase):
 
 - [Overview of OpenLibrary Service Architecture](https://openlibrary.org/about/architecture)
 
-## Running tests
-
-Open Library tests can be run using pytest (py.test).
-
-Inside vagrant, go to the application base directory:
-
-        cd /openlibrary
-        make test
-
-### Integration Tests
```

Comment:
I think we can probably add these back. Or maybe have a link in the Wiki given that we do have some documentation regarding Testing in the Wiki.